### PR TITLE
Make the used types compatible with PHP 7.4

### PR DESCRIPTION
**Pull request by tstauer**
Some return return types and type hints are not compatible with PHP 7.4 despite PHP 7.4 is supported according to the file  "composer.json".

### CODE REVIEW

File: src/FileManager.php [hunk 1]
```diff
@@ -156,7 +156,7 @@ private function fileExist() : bool {
      *
      * @return array|string
```
Comment (igorhrcek) [inline src/FileManager.php:157]: This should also be amended

File: src/FileManager.php [hunk 2]
```diff
@@ -214,7 +214,7 @@ private function findInFile(string $needle) : int|bool {
      *
      * @return array|bool
```
Comment (igorhrcek) [inline src/FileManager.php:215]: This should also be amended

File: src/SubCommands/SubCommand.php [hunk 3]
```diff
@@ -98,11 +98,11 @@ private function setFilePath() : string {
...
 
     /**
-     * Reads rule template file. Depending on output type, returns string or an array
+     * Reads rule template file. Depending on output type, returns an array
      *
-     * @return string|array
+     * @return array
      */
-    private function setRuleContent() : string|array {
+    private function setRuleContent() : array {
```
Comment (igorhrcek) [inline src/SubCommands/SubCommand.php:105]: This method can return both string and an array, just remove multiple types as output type.

**Review by igorhrcek — CHANGES_REQUESTED** [review]
Additional changes are required

File: src/FileManager.php [hunk 4]
```diff
@@ -203,7 +203,7 @@ public function extractRuleBlock(string $marker) : array {
      *
      * @param string $needle
      *
-     * @return int|bool
+     * @return bool|int|string
```
Comment (DanielRuf) [inline src/FileManager.php:206]: Union types are only supported in PHP 8 ;-)

https://stackoverflow.com/a/37033355/753676

Comment (tstauer) [reply src/FileManager.php:206]: Hi there,
it's correct that PHP 8 added support for union types. You are commenting on PHPDoc, though. This is treated like a comment by PHP.

This means the PHP code will run using both PHP 7.4 and 8.0. An IDE like PhpStorm can use the PHPDOC to offer hints to the developers.

Please note I removed the return type of the function in the PHP code due to the compatibility with PHP 7.4.

Comment (DanielRuf) [reply src/FileManager.php:206]: Yes, I'm aware of that. Just wanted to note that the syntax was not compatible, which was fixed in https://github.com/igorhrcek/wp-cli-secure-command/commit/04d9265c0f18ca5d1e9ba2e61ce6afddc1e874f1 =)

**Pull request merged**